.github/workflows/tests.yml

```diff
@@ -258,6 +258,16 @@ jobs:
           path: test-apps/e2e/test-results/**/trace.zip
           retention-days: 1
 
+      # Add the Trunk Analytics Uploader step
```

Comment:
Could we add this to front-end tests too? Or is the plan to just start with e2e and see how it goes?